app/src/main/res/layout/art_object_in_list.xml

```diff
@@ -1,30 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:padding="10dp"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
               android:layout_width="match_parent"
-              android:layout_height="match_parent">
-
-    <de.hdodenhof.circleimageview.CircleImageView
-        android:id="@+id/art_object_view_in_list_thumb"
-        android:layout_width="96dp"
-        android:layout_height="96dp" />
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:textAppearance="?android:attr/textAppearanceMedium"
-        android:text="Medium Text"
-        android:id="@+id/art_object_view_in_list_author"
-        android:layout_alignParentTop="true"
-        android:layout_toEndOf="@+id/art_object_view_in_list_thumb"/>
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:textAppearance="?android:attr/textAppearanceMedium"
-        android:text="Medium Text"
-        android:id="@+id/art_object_view_in_list_name"
-        android:layout_below="@+id/art_object_view_in_list_author"
-        android:layout_toEndOf="@+id/art_object_view_in_list_thumb"/>
-
-</RelativeLayout>
+              android:layout_height="match_parent"
+              android:padding="2dp">
+
+    <android.support.v7.widget.CardView
```

Comment:
I think that some margins for the cardviews will improve the visual look of the whole list